IPython/core/ultratb.py

```diff
@@ -382,8 +381,6 @@ def _format_traceback_lines(lnum, index, lines, Colors, lvals=None,  _line_forma
     i = lnum - index
 
     for line in lines:
-        line = py3compat.cast_unicode(line)
```

Comment:
Careful, this one *isn't* a no-op on Python 3. `cast_unicode` will check for bytes on both Python versions.